website/docs/d/data_git_repository.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_git_repository"
+description: |-
+  Use this data source to access information about an existing Git Repository within Azure DevOps.
+---
+
+# Data Source: azuredevops_git_repository
+
+Use this data source to access information about a **single** (existing) Git Repository within Azure DevOps.
+To read informations about **multiple** Git Repositories use the data source [`azuredevops_git_repositories`](data_git_repositories.html)
```

Comment:
Awesome to have documentation here; thanks for adding it! The feature looks promising. - Just a little typo: "information" should not have an "s", as it is uncountable (https://dictionary.cambridge.org/grammar/british-grammar/information).

Reply:
Corrected